src/models/formFields.ts

```diff
@@ -0,0 +1,17 @@
+export interface FormFields {
+  username: string;
+  name: string;
+  email: string;
+  age: number;
+  skills: any;
+  commitment: number;
+  additionalInfo: string;
```

Comment:
the docs says the field is called "extraInfo" I think having that aligned with backend is better.

Reply:
this is an optional field